### PR TITLE
fix(windows): prevent path transformation of arguments to crane mutate

### PR DESCRIPTION
**Pull request by alexeagle**
Tested manually on windows.

### CODE REVIEW

File: oci/private/image.bzl [hunk 1]
```diff
@@ -174,6 +174,14 @@ def _oci_image_impl(ctx):
         inputs = depset(transitive = inputs_depsets),
         arguments = [args],
         outputs = [output],
+        # Don't convert arguments like --entrypoint=/some/bin to --entrypoint=C:/msys64/some/bin
```
Comment (thesayyn) [inline oci/private/image.bzl:177]: nit: we should only set this on windows?

**Review by thesayyn — APPROVED** [review]

File: oci/private/image.bzl (hunk 1 above)
Comment (alexeagle) [reply oci/private/image.bzl:177]: sure, I'll make it conditional

Comment (alexeagle) [reply oci/private/image.bzl:177]: actually I think I could move it into the .bat wrapper, will test that

Comment (thesayyn) [reply oci/private/image.bzl:177]: that'd be awesome actually.

Comment (thesayyn) [reply oci/private/image.bzl:177]: instead of being part of action cache key, it can be a runtime detail.

Comment (alexeagle) [reply oci/private/image.bzl]: I failed to get that to work, it messes with something and causes

`ERROR: C:/users/alex/documents/github/rules_oci/e2e/smoke/BUILD.bazel:28:12: OCI Tarball //:tar failed: (Exit 1): wrap_tar.bat failed:tar.exe: could not chdir to '/tmp/tmp.Ekhtep7ycr'`

**Review by f0rmiga — APPROVED** [review]

**Pull request merged**